rose/server/game.py

```diff
@@ -9,6 +9,8 @@
 from . import player
 from . import score
 
+FORMAT = '%(asctime)-15s %(levelname)s [%(name)-8s] %(message)s (%(funcName)s:%(lineno)d)'
+logging.basicConfig(format=FORMAT)
```

Comment:
We don't need to add logging.basicConfig - this is done once in rose/server/main.py, rose/client/main.py, and rose-admin.